.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [main]
+    paths-ignore: ["*.md"]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-dotnet@v4
+        with:
+          source-url: ${{ secrets.NUGET_GSOFTDEV_FEED_URL }}
```

Comment:
Looks like IT has created new organization secrets, but the naming is inconsistant. Still works, though

Reply:
Just to be sure you know, this one is to publish preview pacages that's why they created this new secret. Agreed it's not consistent with `GSOFT_NUGET_API_KEY`